dbfit-java/sqlserver/src/main/java/dbfit/environment/SqlServerEnvironment.java

```diff
@@ -133,6 +135,16 @@ protected String parseCommandText(String commandText) {
     // private static string[] GuidTypes = new string[] { "UNIQUEIDENTIFIER" };
     // private static string[] VariantTypes = new string[] { "SQL_VARIANT" };
 
+    private String objectDatabasePrefix(String dbObjectName) {
+ 
```

Comment:
In the existing code seems we don't have such blank lines after the method header. And right now to me it looks a bit better without this extra line.